virttest/utils_package.py

```diff
@@ -88,7 +88,7 @@ def operate(self, timeout, default_status):
                 need = True
             if need:
                 cmd = self.cmd + pkg
-                if self.session.cmd_status(cmd, timeout):
+                if self.session.cmd_status(cmd, timeout, 2):
```

Comment:
suggest interval value can pass in by adding one additional parameter interval with default  value = 2 to operate method

Reply:
```python
def operate(self, timeout, default_status, internal_timeout=2)
    ...
    if self.session.cmd_status(cmd, timeout, internal_timeout):
    ...
```